elki/src/test/java/elki/math/statistics/dependence/MCDEDependenceMeasureTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * This file is part of ELKI:
+ * Environment for Developing KDD-Applications Supported by Index-Structures
+ *
+ * Copyright (C) 2019
+ * ELKI Development Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package elki.math.statistics.dependence;
+
+import elki.utilities.ELKIBuilder;
+import elki.utilities.datastructures.arraylike.NumberArrayAdapter;
+import elki.utilities.datastructures.arraylike.DoubleArrayAdapter;
+import elki.utilities.random.RandomFactory;
+import elki.math.statistics.tests.mcde.*;
+import java.util.Random;
+
+import static elki.math.statistics.dependence.MCDEDependenceMeasure.Par.TEST_ID;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static elki.math.statistics.dependence.MCDEDependenceMeasure.Par.M_ID;
+
+import org.junit.Test;
+
+/**
+ * @author Alan Mazankiewicz
+ * @author Edouard Fouché
+ */
+
+public class MCDEDependenceMeasureTest {
+
+  @Test public void runTest() {
+
+    // independent
+    double[] indi1 = new double[1000];
+    double[] indi2 = new double[1000];
+
+    RandomFactory rnd = new RandomFactory(0);
+    Random generator = rnd.getRandom();
+
+    for(int i = 0; i < 1000; i++) {
+      indi1[i] = generator.nextDouble();
+      indi2[i] = generator.nextDouble();
+    }
+
+    // linear
+    double[] lin1 = new double[1000];
+    double[] lin2 = new double[1000];
+
+    for(int i = 0; i < 1000; i++) {
+      lin1[i] = i;
+      lin2[i] = i * 2;
+    }
+
+    // linear with small noise
+    double[] small_noise_lin1 = new double[1000];
+    double[] small_noise_lin2 = new double[1000];
+
+    for(int i = 0; i < 1000; i++) {
+      small_noise_lin1[i] = i;
+      small_noise_lin2[i] = (i * 2) + generator.nextGaussian() * 100;
+    }
+
+    double[] more_noise_lin1 = new double[1000];
+    double[] more_noise_lin2 = new double[1000];
+
+    for(int i = 0; i < 1000; i++) {
+      more_noise_lin1[i] = i;
+      more_noise_lin2[i] = (i * 2) + generator.nextGaussian() * 500;
+    }
+
+    DoubleArrayAdapter adapter = DoubleArrayAdapter.STATIC;
+    MWPTest mwpTest = MWPTest.STATIC;
+
+    MCDEDependenceMeasure mwp = new ELKIBuilder<>(MCDEDependenceMeasure.class) //
+        .with(M_ID, 1000)
```

Comment:
For consistency, please use `MCDEDependenceMeasure.Par.M_ID` instead of the static import. Also add ` //` to the end of the lines to prevent automatic code formatting to join them.